README.md

```diff
@@ -43,4 +43,87 @@ If you'd like to try out the plugin pre-release, you can build it from source an
+
+![develop on Kubernetes console window](docs/images/console-skaffold-build.gif)
+
+Once the build completes, the image is pushed and deployment starts, the console begins to stream logs from your Kubernetes deployment:
+
+![Spring Boot logs from Kubernetes deployment](docs/images/console-spring-boot-init.gif)
+
+As you can see, Spring Boot application initializes and launches built-in web server. Be default, Spring Boot web server uses port 8080 to serve the content. The plugin and Skaffold make sure you don’t have to worry about accessing the deployment via remote addresses - all declared container ports are port-forwarded automatically!
+
+![automatic port-forwarding](docs/images/auto-port-forward-hello-world.png)
+
+Navigate your browser to `localhost:8080` to access the Spring Boot application running on your Kubernetes cluster. Alternatively, use `curl` command to interact with the application:
+
+<img src="docs/images/browser-root.png" alt="browser showing root page of the application" width="350"/> 
+
+```
+$ curl localhost:8080
+Hello, World of Kubernetes with IntelliJ!
+```
+
+You can check the details of the Kubernetes deployment and service using standard Kubernetes CLI commands (`kubectl get deploy`, etc.) or using Kubernetes dashboard for your Kubernetes cluster. The Kubernetes resources for the project are located in `k8s` directory - there is one deployment and one service YAML file.
+
+Now, let’s add more features to our Spring Boot project and see how they get deployed to your Kubernetes cluster without stopping and removing the deployment, manually building and tagging the image, and updating the cluster. Open `HelloController.java` file from `src` and add a new HTTP request mapping:
+
+```java
+   @RequestMapping("/greeting")
+    public String greeting(@RequestParam(value="name", defaultValue="World") String name) {
+        return String.format("Hello from Kubernetes with IntelliJ, %s!", name);
+    }
+```
+
+Save the changes (`Ctrl-S`) or build the project (use `Build -> Build Project` menu or the toolbar icon). The plugin and Skaffold pick up the changes, re-build the project, image and deploy the updated image to your Kubernetes cluster. You can watch the progress and deployment logs in the console window. Once our updated Spring Boot application initializes, we can test our application running on Kubernetes is still accessible and the new feature is immediately available:
+
+![browser showing new greeting page of the application](docs/images/browser-greeting.png)
+
+```
+$ curl localhost:8080/greeting?name=User
+Hello from Kubernetes with IntelliJ, User!
+```
+
+You can continue adding and debugging new features and have them built as an image and deployed to your Kubernetes cluster from your IDE on every change. Once you are finished, click stop to finish continuous development on Kubernetes session. The plugin and Skaffold delete all Kubernetes resources used for the development session.
```

Comment:
"click stop to finish..." ->
"click `stop` to end the continuous development session."

"The plugin and Skaffold delete" -> "The plugin deletes"